casatables/src/lib.rs

```diff
@@ -542,6 +623,16 @@ pub enum TableOpenMode {
     Create = 3,
 }
 
+pub enum TableCreateMode {
+    // create table
```

Comment:
Enum variants should ideally be documented with triple-slash `///` docstrings (although again I'll fully admit that I have not been good about writing this level of documentation myself).

Reply:
fixed!